packages/api/src/controllers/asset.ts

```diff
@@ -146,15 +150,25 @@ async function validateAssetPayload(
   };
 }
 
-function validateAssetPlaybackPolicy(playbackPolicy: Asset["playbackPolicy"]) {
+function validateAssetPlaybackPolicy(
+  playbackPolicy: Asset["playbackPolicy"],
+  playbackId?: string,
+  createdAt?: number
```

Comment:
I don't like passing the params, I would have passed the asset itself but in the validateAssetPayload I have no asset yet, will make it in a better way later

Reply:
Hmm yeah, let's refrain from refactoring this too much until we decide we are going through with the feature, not just the poc